.github/workflows/build-test.yml

```diff
@@ -141,20 +174,41 @@ jobs:
   javascript:
     name: javascript
     timeout-minutes: 15
-    needs: smoke
+    needs: cache-gremlin-server-docker-image
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest]
+        # Windows Disabled until Linux containers are supported on Windows runners: https://github.com/actions/virtual-environments/issues/252
+        # os: [ubuntu-latest, windows-latest]
+        os: [ubuntu-latest]
     steps:
       - uses: actions/checkout@v3
       - name: Set up JDK 11
         uses: actions/setup-java@v3
         with:
           java-version: '11'
           distribution: 'temurin'
+      - name: Get Cached Server Base Image
+        if: matrix.os == 'ubuntu-latest'
+        uses: actions/cache@v3
+        id: gremlin-server-test-docker-image
+        with:
+          path: |
+            ./gremlin-server/*
+            ~/.m2/repository/org/apache/tinkerpop/*
+          key: ${{ github.sha }}
+      - name: Download Server Base Image
+        if: matrix.os == 'windows-latest'
```

Comment:
(nitpick) Since you have commented out the `windows-latest` completely above in line 182, I suggest this step can also be commented out completely.

Reply:
Done.